Backend/dev/mock-servers/server.py

```
@@ -249,6 +260,187 @@ def _mock_override(self, body):
+                conn.autocommit = True
+                cur = conn.cursor()
+                cur.execute(
+                    "UPDATE atlas_app.purchased_pass SET status = %s, updated_at = NOW() WHERE id = %s",
+                    (status, pass_id)
+                )
+                result = {"ok": True, "rowcount": cur.rowcount}
+                cur.close()
+            finally:
+                conn.close()
+            return self._json(result)
+        except Exception as e:
```

Ruff (0.15.10)

[warning] Do not catch blind exception: `Exception`

(BLE001)